source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -74,6 +74,18 @@ def to_int(s):
     return s
 
 
+def _ts_to_dt(ts: int) -> str:
+    """
+    Converts a millisecond UNIX timestamp to a date-time formatted string.
+    """
+    return datetime.fromtimestamp(ts, tz=UTC).isoformat(' ')
+
+def _dt_to_ts(dt_str: str) -> int:
+    """
+    Converts a date-time formatted string to a millisecond UNIX timestamp.
+    """
+    return int(datetime.fromisoformat(dt_str).timestamp())
```

Comment:
I think this is actually going to return seconds - the float result has the integer part as seconds, and the fractional part as microseconds. We have [similar helpers](https://github.com/estuary/connectors/blob/75c2d46cce5260076043770ee5ae8c1a3be3863c/source-hubspot-native/source_hubspot_native/api.py#L893-L897) elsewhere that account for this to get millisecond epoch times.

Reply:
You're right, thanks for catching that. I was wrong in that function description. Zendesk's timestamps are in seconds, not milliseconds. I'll update the function description to say "...to a UNIX timestamp in seconds". 

Also, the reason I stored the state as a date-time formatted string instead of an actual `datetime` objects was because when I retrieved the state after restarting the task, the retrieved state wasn't a `datetime` object but was the ISO formatted string. I'll update the helper function names to be clearer that they're accepting/returning a string and not a `datetime` object.